modules/menu/css/lib/menu.scss

```diff
@@ -1,15 +1,15 @@
 .wdc-menu {
   @include wdc-type();
   @include wdc-type-body();
-  @include wdc-depth-3();
+  @include wdc-depth-2();
   display: inline-block;
   background-color: $_wdc-menu-bg-color;
-  padding: $wdc-spacing-s;
   border-radius: 3px;
   border: 1px solid $_wdc-menu-border-color;
   position: relative;
   width: max-content;
-  max-width: $_wdc-menu-max-width;
+  -webkit-font-smoothing: auto;
```

Comment:
What's the correct way to override font smoothing here? I had to do this in order to get the CSS styles to match the React styles. Currently, I'm overriding styles from the `wdc-type` mixin directly in this Sass file, but I'm not sure that's the right way to go about this.

Reply:
I think this may be incorrect in the React implementation then. We generally try to use `*-font-smoothing: subpixel-antialiased;` for everything.